src/ui-test/test-repo/flask-hello-world/README.md

```diff
@@ -0,0 +1,3 @@
+# flask-hello-world
```

Comment:
We should change the README here to give context to what this is used for.

In this project, it's used to simulate a real user when testing.

Reply:
Updated.